rubocop.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
   s.name = 'rubocop'
   s.version = RuboCop::Version::STRING
   s.platform = Gem::Platform::RUBY
-  s.required_ruby_version = '>= 2.2.0'
+  s.required_ruby_version = '>= 2.2.2'
```

Comment:
Psych v3.1 supports Ruby 2.2.2 or higher. 
https://github.com/ruby/psych/blob/v3.1.0/psych.gemspec#L50

So we need to drop supporting of Ruby 2.2.1 and 2.2.0.
I think it is not a big problem. RuboCop still supports Ruby 2.2.x. And I think we'll drop Ruby 2.2 soon.

Reply:
I agree, but I worried about the following. 
Dropping the Ruby version 2.2.1 and 2.2.0 may be better opened as a separate PR. And you can also edit the code using Psych at that time.
https://github.com/rubocop-hq/rubocop/pull/6733/files#diff-e2fc41e8fa0504d203570a8d822495afL186

I think that the opportunity for users to know written in CHANGELOG will increase than to investigate PR.

Reply:
Thank you for your comment. I'll create a pull request to drop Ruby 2.2.1 and 2.2.0. :+1:

Reply:
:tada:

Reply:
I opened a pull request. #6766 
Thank you!